src/views/components/landing-top.html

```diff
@@ -4,303 +4,174 @@
 <div class="c-landing-top">
 
 	<div class="content">
-		<div class="container">
-
-			<section class="heading">
-				<span>Fast. Reliable. Automated.</span>
-				<h1>A free CDN for open source projects</h1>
-				<p>Optimized for JS and ESM delivery. Full NPM and GitHub integration. Works with all web formats.</p>
-			</section>
-
-			<section class="stats">
-				<div>
-					<h3>{{_.formatHits(networkStats.hits.total)}} billion request</h3>
-					<span>last month</span>
-				</div>
-				<div>
-					<h3>over {{_.formatMiB2TiB(networkStats.megabytes.total)}} TiB</h3>
-					<span>last month</span>
-				</div>
-			</section>
+		<section class="heading">
+			<span>Fast. Reliable. Automated.</span>
+			<h1>A free CDN for open source projects</h1>
+			<p>Optimized for JS and ESM delivery. Full NPM and GitHub integration. Works with all web formats.</p>
+			<span>Operating since 2012.</span>
+		</section>
+
+		<section class="stats">
+			<div>
+				<h3>{{_.formatHits(networkStats.hits.total)}} billion request</h3>
+				<span>last month</span>
+			</div>
+			<div>
+				<h3>over {{_.formatMiB2TiB(networkStats.megabytes.total)}} TiB</h3>
+				<span>last month</span>
+			</div>
+		</section>
 
 
 		<section class="docs">
-			<div class="docs-tabs">
```

Comment:
Why are we changing from `docs-tabs` to `docs_tabs` in some places? I believe we used hyphens everywhere until now.

Reply:
Easier to read when nested elements separated by dash and the name parts with hyphens